docker-compose-production.yml

```diff
@@ -53,7 +53,7 @@ services:
       - "sdschedule-letsencrypt:/etc/letsencrypt:ro" # must keep sercret
       - "./web/letsencrypt/www:/var/www/_letsencrypt:ro"
     restart: always
-    command: sh -c "exec nginx -g 'daemon off;'; while true; do nginx -s reload; sleep 12h; done"
+    command: sh -c 'while :; do sleep 12h; nginx -s reload; done & nginx -g "daemon off;"'
```

Comment:
Our service is running an infinite loop in the background here.

I don't know if this is the correct thing to do. Help requested.

Reply:
FYI, in the previous command, the while loop is never executed since the `nginx` command will not exit

Reply:
The expiry date of let’s encrypt certificates is 90 days.

So we need to renew it at least every 90 days (not harmful to be more frequent)

And for each renewal, Nginx needs to be reloaded to apply the newly renewed certificates.

I set it to be 24h and 12h now to test. Can be set longer after things go stable.

Reply:
Ok sure, sounds good!